frontend/packages/integration-tests-cypress/tests/crud/k8-openshift-cruds.spec.ts

```diff
@@ -143,6 +145,9 @@ describe('Kubernetes resource CRUD operations', () => {
             namespaced ? `ns/${testName}` : 'all-namespaces'
           }?kind=${kind}&q=${testLabel}%3d${testName}&name=${name}`,
         );
+
+        // filter should have 3 chip groups: resource, label, and name
+        cy.get("[class='pf-c-toolbar__item pf-m-chip-group']").should('have.length', 3);
```

Comment:
Ideally this selector would move to a view file.

Reply:
> @dtaylor113 When you say reload, do you mean the entire page reloaded in the browser? Or just that the page updated? We might want to dig into this and see if there is a bug here. We do have a bug tracking problems with the name filter.

I 'think' it was the page updating.  It initially loaded with name filter in place, then I saw the three dot (table?) loading indicators, then the table with the single resource reappeared.

Reply:
ok, fixed